apps/jubmoji-quest/src/components/cards/CollectionCard.tsx

```diff
@@ -2,23 +2,22 @@ import React, { useState } from "react";
 import { Icons } from "../Icons";
 import { classed } from "@tw-classed/react";
 import { Button } from "../ui/Button";
-
-interface Quest {
-  label: string;
-  url: string;
-}
+import Link from "next/link";
 
 interface CollectionCardProps {
-  icon: string; // jubmoji icon
-  label: string; // meaning of the jubmoji
-  cardBackImage?: string;
+  icon: string;
+  label: string;
   edition: number;
   owner: string;
-  location?: string;
-  questName: string;
-  linkToQuest?: string;
+  cardBackImage?: string;
   actions?: React.ReactNode;
-  quests?: Quest[];
+  quests?: {
```

Comment:
This would be a `JubmojiQuestPreview` type, I'm a little hesitant to over-type things where it's only going to be used once. This is different from the `JubmojiQuest` type as it doesn't need many of the parameters there.